assets/javascripts/discourse/templates/components/campaign-banner.hbs

```diff
@@ -13,22 +13,21 @@
       background-size: cover;
       background-repeat: no-repeat;' ''}}">
       {{#if isGoalMet}}
-         <h2 class="campaign-banner-info-header">{{i18n 'discourse_subscriptions.campaign.success_title'}}</h2>
+        <h2 class="campaign-banner-info-header">{{i18n 'discourse_subscriptions.campaign.success_title'}}</h2>
         <p class="campaign-banner-info-description">{{i18n 'discourse_subscriptions.campaign.success_body'}}</p>
       {{else}}
         <h2 class="campaign-banner-info-header">{{i18n 'discourse_subscriptions.campaign.title'}}</h2>
         <p class="campaign-banner-info-description">{{i18n 'discourse_subscriptions.campaign.body'}}</p>
+        {{#if product}}
+          {{#link-to "s.show" product disabled=product.subscribed class="btn btn-primary campaign-banner-info-button"}}
```

Comment:
I believe I've said this before but I think it's technical debt that the route is called `s`. I'm fine with the path being `/s` but the route should be called something more descriptive like `subscriptions` and we should fix this in the future.

Reply:
You got it -- I'll add it to the list to refactor in the future.